lib/pact.ex

```diff
@@ -110,6 +120,21 @@ defmodule Pact do
     {:stop, :normal, :ok, container}
   end
 
+  defp create_fake_module_name(name, pid) do
+    process_id_as_string = IO.iodata_to_binary(:erlang.pid_to_list(pid))
```

Comment:
I thought about that too, but I'm not sure. Is the variable name not clear enough? Maybe I could extract a function to make it clear?